app/views/suggestions/index.erb

```diff
@@ -0,0 +1,8 @@
+<a href="/users/current_user.id"> Profile </a>
+<br>
+<br>
+<% if @suggestions %>
```

Comment:
Heads up: even if `Suggestion.all` returns no suggestions, it will still return an empty collection, which is truthy. Thus, this `if` conditional will always execute, even when there are no suggestions in `@suggestions`. (Just like how `Enumerable#select` returns an empty array (which is truthy!) even if it finds no matches.)